src/styles/index.scss

```diff
@@ -1,3 +1,16 @@
-body {
+@import './card';
+@import './stars';
+@import './moto_styles';
+@import './catalog';
+@import './utils/variables';
+
+* {
+  box-sizing: border-box;
+  transition-duration: 300ms;
```

Comment:
It is better not to set a transition for the entire page, it affects the probability.